prompt2model/dataset_generator/prompt_based.py

```diff
@@ -20,6 +20,7 @@
 from prompt2model.utils import (
     API_ERRORS,
     APIAgent,
+    api_tools,
```

Comment:
Why not `API_TOOLS`?

Reply:
This is a module, not a variable, so I think lower-case is standard.